src/pages/addons.tsx

```diff
@@ -37,21 +37,14 @@ const addons = [
                         icon: IconAdvanced,
                     },
                     {
-                        name: 'Enforce event schemas',
+                        name: 'Library of pre-built templates',
```

Comment:
This now sounds a bit drab... and really doesn't sell the benefits. I think we should an entry about "automating your other cloud services". Feel free to give it a go or I'll do it in follow up